src/Compilers/CSharp/Portable/Binder/DecisionDagBuilder.cs

```diff
@@ -11,6 +11,8 @@
 using Microsoft.CodeAnalysis.PooledObjects;
 using Roslyn.Utilities;
 
+#nullable enable
```

Comment:
@gafter I reviewed all the files in this PR except for this one. If you need someone to finish the review while I'm away, I'm fine with that (they should focus on this file)